aguskoll/target-app
# Feature/google maps in home screen

### What's in

The home screen layout.
The map in the home screen, with a yellow marker in an example location. 
The access to the profile screen from the home. 

### How it looks

https://drive.google.com/file/d/1-ORbmokHoeiAVBDGNsMqK2Q9pJNsDRTv/view?usp=sharing

### How to test

1. Run this branch and log in.
2. Verify that the map is loaded.
3. Verify that you see a yellow marker with a circle around
4. Verify that you can go to the profile screen.
